resolwe_bio/descriptors/super_sample.yml

```diff
@@ -0,0 +1,263 @@
+- slug: super_sample
+  name: Super sample annotation
+  description: Sample meta data compliant with GEO and ENCODE databases.
+  version: 0.0.1
```

Comment:
Let's make the first one 1.0.0, as it then looks more production ready to the end-users.